src/webpack.config.ts

```diff
@@ -345,6 +345,9 @@ function webpackConfig(args: Partial<BuildArgs>) {
 				{ test: /@dojo\/.*\.js$/, enforce: 'pre', loader: 'source-map-loader-cli', options: { includeModulePaths: true } },
 				{ test: /src[\\\/].*\.ts?$/, enforce: 'pre', loader: '@dojo/webpack-contrib/css-module-dts-loader?type=ts&instanceName=0_dojo' },
 				{ test: /src[\\\/].*\.m\.css?$/, enforce: 'pre', loader: '@dojo/webpack-contrib/css-module-dts-loader?type=css' },
+				{ test: /\.js(x)?$/, loader: '@dojo/webpack-contrib/static-build-loader', options: {
```

Comment:
I don't believe we will ever have `.jsx` files emitted... The TypeScript compiler will only ever emit `.js` files for us.  If for some reason we did, we wouldn't want to touch them I suspect.

Reply:
Ok. I added the loader to the existing loader config for all `js` files and removed this separate entry entirely.